test/ingestors/zora-mint.test.ts

```diff
@@ -0,0 +1,101 @@
+import { expect } from 'chai';
+
+import { ZoraIngestor } from '../../src/ingestors/zora';
+import { EVMMintInstructions } from '../../src/lib/types/mint-template';
+import { MintTemplateBuilder } from '../../src/lib/builder/mint-template-builder';
+import { mintIngestorResources } from '../../src/lib/resources';
+import { basicIngestorTests } from '../shared/basic-ingestor-tests';
+
+describe('zora-base-mint', function () {
+
+  basicIngestorTests(new ZoraIngestor(), mintIngestorResources(), {
```

Comment:
This mint is expired now, and I can't figure out the blockhash that it's available at.

For simplicity, these values will pick a new mint, and then fix the blockhash of the simulation so that it will always succeed.

```
  basicIngestorTests(
    new ZoraIngestor(),
    mintIngestorResources(),
    {
      successUrls: ['https://zora.co/collect/base:0x923b382f3205c26efe706c53d69f71f0817aa954/1'],
      failureUrls: ['https://example.com'],
      successContracts: [
        { chainId: 8453, contractAddress: '0x1e1ad3d381bc0ccea5e44c29fb1f7a0981b97f37', tokenId: '1' },
      ],
      failureContracts: [{ chainId: 8453, contractAddress: '0x6140F00e4Ff3936702E68744f2b597885464cbB', tokenId: '1' }],
    },
    {
      8453: '0x10B4CB9',
    },
  );
```